beater/authorization/apikey.go

```diff
@@ -0,0 +1,149 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
since we will also need a bunch of `apikey` functionality for the subcommand, it might make sense to move this file to a top level `apikey` package.

Reply:
Not sure why that would be a reason to move it to top level. I assume you are adding the tooling to the `cmd` package or as a sub-package `cmd/apikey`. There is no harm in importing the `beater` package from there (it is already imported for the other commands).